charts/s1-agent/templates/_helpers.tpl

```diff
@@ -187,6 +187,26 @@ Generate certificates for helper secret
 {{- dict "tls.crt" $tlsCert "tls.key" $tlsKey "ca.crt" $caCert | toYaml -}}
 {{- end -}}
 
+{{- define "helper_token.secret.create" -}}
+{{- empty .Values.secrets.helper_token | ternary "true" "" }}
+{{- end -}}
+
+{{- define "helper_token.secret.name" -}}
+{{- if include "helper_token.secret.create" . }}
+{{- include "helper.fullname" . -}}
```

Comment:
is this needed?

Reply:
As same as site key and other secrets, we allow customers to set their own token secret and just set the name. So in that case we don't create a token secret, just using the one that was set by the customer.